javascript/frameworks/cap/src/cqlinjection/CqlInjection.ql

```diff
@@ -12,38 +12,27 @@
 
 import javascript
 import DataFlow::PathGraph
-import semmle.javascript.security.dataflow.SqlInjectionCustomizations::SqlInjection
-import advanced_security.javascript.frameworks.cap.CQL
-import advanced_security.javascript.frameworks.cap.RemoteFlowSources
+import advanced_security.javascript.frameworks.cap.CAPCqlInjectionQuery
 
-class CqlIConfiguration extends TaintTracking::Configuration {
-  CqlIConfiguration() { this = "CqlInjection" }
-
-  override predicate isSource(DataFlow::Node source) { source instanceof RemoteFlowSource }
-
-  override predicate isSink(DataFlow::Node sink) { sink instanceof CQLSink }
-
-  override predicate isSanitizer(DataFlow::Node node) {
-    super.isSanitizer(node) or
-    node instanceof Sanitizer
-  }
-
-  override predicate isAdditionalTaintStep(DataFlow::Node pred, DataFlow::Node succ) {
-    //string concatenation in a clause arg taints the clause
-    exists(TaintedClause clause |
-      clause.getArgument() = pred.asExpr() and
-      clause.asExpr() = succ.asExpr()
-    )
-    or
-    //less precise, any concat in the alternative sql stmt construction techniques
-    exists(ParseCQLTaintedClause parse |
-      parse.getAnArgument() = pred and
-      parse = succ
-    )
-  }
+DataFlow::Node getQueryOfSink(DataFlow::Node sink) {
```

Comment:
Could we create a `CQLSink` class with a `getQuery` predicate, instead of duplicating this logic between the `isSink` definition for the dataflow configuration and this predicate.